charts/sonarqube-dce/Chart.yaml

```diff
@@ -70,6 +70,6 @@ dependencies:
     repository: https://raw.githubusercontent.com/bitnami/charts/pre-2022/bitnami
     condition: postgresql.enabled
   - name: ingress-nginx
-    version: 4.7.1
+    version: 4.8.3
```

Comment:
Hey @jCOTINEAU,

shall we log this update in the change list? I think we haven't always done this in the past, but we should. WDYT?

Reply:
that is a very good catch doing it now